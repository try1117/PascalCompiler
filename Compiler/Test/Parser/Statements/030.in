program test;
var
  a: integer;
  b: double;
  c: char;
  d: string;
begin
  read(a, b, c, d);
  write(1.0 * a, a + 5 * 5, 'qwerty' + d);
end.
