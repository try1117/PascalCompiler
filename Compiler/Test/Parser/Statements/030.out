test : function()
   resultType : Nil

test declarations:
   a : Integer

   b : Double

   c : Char

   d : String

|-- Statements
|            |-- Read
|            |      |-- a
|            |      |-- b
|            |      |-- c
|            |      --- d
|            --- Write
|                    |-- *
|                    |   |-- 1.000000
|                    |   --- a
|                    |       |-- a
|                    |-- +
|                    |   |-- a
|                    |   --- 25
|                    --- +
|                        |-- 'qwerty'
|                        --- d

